### DISCUSSION TITLE
Unmarked

### DISCUSSION THREAD

**Discussion by jeffreykolb** [matsn0w/IIS-EmergencyLS-ELS-FiveM#139] [Q&A] [answered]
Hey I have unmarked vehicles on my server, just want to add a pit to the ELS when you turn on the lights. How do I do that?

**Comment by matsn0w** [comment, marked as answer]
Hi there, sorry for my late response!

You can add the pit as a static extra. See the 'Statics' section of the VCF Configurator and also see [the docs](https://github.com/matsn0w/MISS-ELS/blob/main/docs/vcf.md#static-extras-static).

![image](https://github.com/matsn0w/MISS-ELS/assets/15019582/2679e61a-20b6-4b9d-a24a-9e1c5bbc0b82)

EDIT: I'm sorry, I've misread your question. Currently it is not possible to automatically turn the statics on when activating the lights. There is already an open issue for that: https://github.com/matsn0w/MISS-ELS/issues/107
